definitions/ext-gatsby_build/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: GATSBY_BUILD
+
+synthesis:
+  name: gatsbySite
+  identifier: gatsbySite
+  encodeIdentifierInGUID: true
+  conditions:
+    - attribute: service.name
```

Comment:
Can you update this to:

```
conditions:
  - attribute: gatsbySite
```

And no value? (this should fully avoid the EXT-SERVICE rule)

Reply:
done 👯